frontend/src/general/userForm/FormConsole.jsx

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import { Button, Container, TextField } from "@material-ui/core";
+import PropTypes from "prop-types";
+
+export default class FormConsole extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: "",
+      password: ""
+    };
+    this.onSubmit = this.onSubmit.bind(this);
+    this.setUsername = this.setUsername.bind(this);
+    this.setPassword = this.setPassword.bind(this);
+  }
+
+  onSubmit(event) {
+    const { handleSubmit } = this.props;
+    const { username, password } = this.state;
+    event.preventDefault();
+    handleSubmit(username, password);
+  }
+
+  setUsername(username) {
+    this.setState({
+      username
+    });
+  }
+
+  setPassword(password) {
+    this.setState({
+      password
+    });
+  }
+
+  render() {
+    const { buttonName, isValidate, isCreateUser } = this.props;
+    const { password } = this.state;
+    return (
+      <Container component="main" maxWidth="xs">
```

Comment:
Minor: Consider using a dashboard card for consistency of style throughout the application